README.md

```diff
@@ -108,43 +108,86 @@ See [CONFIG.md](CONFIG.md) for details.
 See [Code Samples](https://github.com/hazelcast/hazelcast-nodejs-client/tree/master/code_samples)
 
 
-# Serialization Considerations
+# Serialization
 
-Hazelcast needs to serialize objects in order to be able to keep them in the server memory. For primitive types, it uses Hazelcast native serialization. For other complex types (e.g. JS objects), it uses JSON serialization.
+Hazelcast serializes all your objects into an instance of `Data`. `Data` is the binary representation of an object. When Hazelcast serializes an object into `Data`, it first checks whether the object is an instance of `IdentifiedDataSerializable`, if not it checks if it is an instance of `Portable` and serializes it accordingly. For the following types Hazelcast optimizes the serialization a user can not override this behaviour.
+- A `boolean` type which is a primitive on Node.js can be serialized as a `Boolean` for Java server-side.
+- A character which is a string with length one can be serialized as a `Character` for Java server-side.
+- A `number` type which is a primitive on Node.js can be serialized as `Byte`, `Short`, `Integer`, `Float`, `Double` for Java server-side.
+- A `Long` type which represents a 64 bit two's-complement integer value can be serialized as a `Long` for Java server-side.
+- A `string` type which is a primitive on Node.js can be serialized as a `String` for Java server-side.
+- Arrays of the above types can be serialized as `boolean[]`, `char[]`, `byte[]`, `short[]`, `int[]`, `float[]`, `double[]`, `long[]`, `string[]` for Java server-side respectively. 
```

Comment:
[1] ...`long[]` and `string[]` for Java server side...

[2] To which order does "respectively" refer here?

Reply:
All types are mentioned respectively while reading the header.